helper_dart/lib/src/facet_list.dart

```diff
@@ -90,6 +92,8 @@ abstract class FacetList implements Disposable {
         groupID: FilterGroupID(attribute, operator),
         selectionMode: selectionMode,
         persistent: persistent,
+        eventTracker: searcher.eventTracker,
+        clickEventName: clickEventName ?? 'click$attribute',
```

Comment:
Does the default value match the same logic as insights.js ?

Reply:
Yes, I updated the name. It's no more configurable and matches the web implementation.